backend/prompt_studio/prompt_studio_registry/models.py

```diff
@@ -72,3 +86,14 @@ class PromptStudioRegistry(BaseModel):
         blank=True,
         editable=False,
     )
+    shared_to_org = models.BooleanField(
```

Comment:
@johnyrahul [DOUBT] Does the intermediary table introduced take care of cascade delete here?

Reply:
No @harini-venkataraman  we have handle it. [Refer](https://docs.djangoproject.com/en/2.0/ref/models/relations/#django.db.models.fields.related.RelatedManager.clear)
We have handled this [here](https://github.com/Zipstack/unstract/blob/93eac974ef916bc4151aff34776f98ed7236c721/backend/account/authentication_controller.py#L406)